rules/correct-component-name.js

```diff
@@ -0,0 +1,109 @@
+const { dim, bold } = require('colors');
+const { Rule } = require('../api');
+
+module.exports = class extends Rule {
+    getName() {
+        return 'correct-component-name';
+    }
+
+    checkTwigFile(component) {
+        const { twig } = component.files;
```

Comment:
Please check views and templates.

Reply:
I think, it will be better check views and template in another rule